notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T232-analysis_actuators.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    [autoZmin, autoZmax, autoLateralMax, forces_zero] = getZeroValuesAndLimits(forces, subtractBaseline, baselineT0, baselineT1)
</pre><p>I did something similar in another code and Russell suggested that when returning too many parameters (basically more than 2), we should use a data class instead. It has many advantages with the price of a bit more of coding. Here are his comments <a href="basically more than 2" target="_blank">DM-38198</a> and here is an example of code in the <a href="basically more than 2" target="_blank">random_walk.py</a> SAL Script. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I left this as it is.  I understand what you are saying, but it doesn't seem worth the time and effort.